src/app/users/user.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+//import { Subject } from 'rxjs';
+import { IUser } from './user.interface';
+@Injectable({ providedIn: 'root' })
+export class UserService {
+  // userChanged = new Subject<IUser[]>();
+  constructor(private http: HttpClient) {}
+  //private users: IUser[] = [];
```

Comment:
Remove all the unused  //   commented code.

Reply:
done!!